src/allotropy/parsers/revvity_matrix/revvity_matrix_reader.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+
+from allotropy.named_file_contents import NamedFileContents
+from allotropy.parsers.utils.pandas import (
+    read_excel,
+)
+
+
+class RevvityMatrixReader:
+    SUPPORTED_EXTENSIONS = "xlsx"
```

Comment:
@nathan-stender, was just reviewing back at the instrument documentation. The supported extension should be .CSV actually. The test file we included here must have been converted to .xlsx post-instrument export.

Reply:
Added support for CSV 👍